domain/domain_test.go

```diff
@@ -48,11 +48,19 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
-var _ = Suite(&testSuite{})
+var _ = SerialSuites(&testSuite{})
 
 type testSuite struct {
 }
 
+func (s *testSuite) SetUpSuite(c *C) {
+	testleak.BeforeTest()
```

Comment:
Could we put it to `TestT`? Then we can remove `SetUpSuite` and `TearDownSuite`.

Reply:
If we put it into `TestT`, it will check only one time, not per `testSuite`.

Reply:
I feel no problem. It shouldn't matter if you check once or twice. After all tests are completed, check once for any extra goroutine.